CICD/CICD.cs

```diff
@@ -67,13 +70,27 @@ public partial class CICD : NukeBuild
 
     private Configuration Configuration => GetBuildConfig();
 
-    private IGitRepoService Repo => App.Container.GetInstance<IGitRepoService>();
+    private IGitRepoWrapper Repo => App.Container.GetInstance<IGitRepoWrapper>();
 
     [Parameter("The output directory of where the template workflows should be generated.")]
     private string? WorkflowTemplateOutput { get; set; }
 
     [Parameter("If true, will skip the twitter announcement of a release.")]
```

Comment:
Refactor all lowercase twitter to uppercase Twitter.
```suggestion
    [Parameter("If true, will skip the Twitter announcement of a release.")]
```

Reply:
Great catch!!